apps/lfx-one/src/app/modules/dashboards/components/pending-actions/pending-actions.component.scss

```
@@ -3,6 +3,24 @@
 
 // Both transitions run for 300ms so the longest CSS animation matches PENDING_ACTION_FADE_OUT_MS in
 // @lfx-one/shared/constants and the JS doesn't drop the row before the fade has finished playing.
+
+// Section-level fade-out when all actions are resolved. max-height is slightly longer than opacity
+// so the collapse completes smoothly after the content has already faded.
+// overflow: hidden is scoped to .is-fading only so normal content is never clipped.
+.pending-actions-section {
+  transition:
+    opacity 300ms ease-out,
+    max-height 400ms ease-out;
+  // Generous ceiling; only constrains during animation, never clips at rest (overflow is visible by default).
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)